gradle/libs.versions.toml

```diff
@@ -49,7 +49,7 @@ kover="0.5.0"
 kspTesting="1.4.7"
 
 # Tools
-detekt = "1.18.0"
+detekt = "1.22.0"
```

Comment:
we don't need this property anymore, because we don't reference Detekt anymore from Gradle.

Reply:
We do reference it in the custom detekt rules project